tedana/workflows/tedana.py

```diff
@@ -422,15 +430,21 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     if png:
         LGR.info('Making figures folder with static component maps and '
                  'timecourse plots.')
+        # make figure folder first
+        if not os.path.isdir('figures'):
+        if not op.isdir(op.join(out_dir, 'figures')):
+            os.mkdir(op.join(out_dir, 'figures'))
+
         viz.write_comp_figs(data_oc, mask=mask, comptable=comptable, mmix=mmix,
                             n_vols=n_vols, acc=acc, rej=rej, midk=midk,
-                            empty=ign, ref_img=ref_img)
+                            empty=ign, ref_img=ref_img, out_dir=out_dir,
```

Comment:
Perhaps `out_dir` for these functions should be `op.join(out_dir, 'figures')` (i.e., `ref_img=ref_img, out_dir=op.join(out_dir, 'figures'), ...`

Reply:
I like this, cleans up figure writing a bit